.gitignore

```diff
@@ -50,3 +50,4 @@ jspm_packages
 .history/
 .scannerwork
 lib
+docs
```

Comment:
We already have a docs folder that contains our logo and should not be ignored right ?

Reply:
docs/logo.png -> images/logo.png